its/tests/src/test/java/its/SonarQubeDeveloperEditionTests.java

```diff
@@ -1218,35 +1205,46 @@ void shouldContainExtendedDescription() throws Exception {
         // For some reason, there is an extra line break in the generated HTML
         expected = "<h1>Title\n</h1><strong>my dummy extended description</strong>";
       }
-      assertThat(engine.getActiveRuleDetails(endpointParams(ORCHESTRATOR), serverLauncher.getJavaImpl().getHttpClient(CONNECTION_ID), javaRuleKey("S106"), projectKey).get()
-        .getExtendedDescription())
-          .isEqualTo(expected);
+
+      backend.getConfigurationService().didAddConfigurationScopes(new DidAddConfigurationScopesParams(
+        List.of(new ConfigurationScopeDto(CONFIG_SCOPE_ID, null, true, projectName, new BindingConfigurationDto(CONNECTION_ID, projectKey, true)))));
+
+      await().untilAsserted(() -> assertThat(didSynchronizeConfigurationScopes).isNotEmpty());
```

Comment:
There is little chance of issue here, but you could wait for the exact config scope:
`await().untilAsserted(() -> assertThat(didSynchronizeConfigurationScopes).contains(CONFIG_SCOPE_ID));`

Reply:
Addressed in 48b21ba